Kernel/Devices/Storage/ATA/AHCI/Definitions.h

```diff
@@ -403,6 +415,8 @@ struct [[gnu::packed]] HBA {
     u8 vendor_specific[96];
     PortRegisters port_regs[];
 };
+static_assert(AssertSize<HBA, 0x100>());
+static_assert(__builtin_offsetof(HBA, port_regs[31]) == 0x1080);
```

Comment:
As this can have 32 entries, getting the offset of the 32nd will give you the size, which might be nicer?
Nit, not a must

Reply:
I based all sizes and this offsetof on the specs, and the spec only listed the start end end of each field. But if you want the size instead, I could use entry 32.

Reply:
you do you